util-stats/src/test/java/com/twitter/finagle/stats/StatsReceiverCompilationTest.java

```diff
@@ -23,7 +23,7 @@ public void testNullStatsReceiver() {
 
   @Test
   public void testStatMethods() {
-    InMemoryStatsReceiver sr = new InMemoryStatsReceiver();
+    InMemoryStatsReceiver sr = new InMemoryStatsReceiver(0);
```

Comment:
are these changes necessary since the default argument is zero?

Reply:
They are necessary in Java code.